### DISCUSSION TITLE
querySelectorAll type action to find all matching inside the alpine component only?

### DISCUSSION THREAD

**Discussion by yabdab** [alpinejs/alpine#3759] [1. Help] [answered]
Similar to ```document.querySelectorAll('.div.step')``` , BUT only gather matching nodes inside the current Alpine component?

Is this possible?

**Comment by yabdab** [comment, marked as answer]
I found the answer immediately after asking and thought I would share ...

```this.$el.querySelectorAll('div.step');```

This happens every time , ask on a public forums and it magically pops into my head 😖

**Comment by ekwoka** [comment]
Yup, one mager benefit of QS and QSA is that they exist on the Element interface, not just the Document interface :)
